cpp/src/converters/COOtoCSR.cuh

```diff
@@ -111,8 +111,10 @@ void fill_offset(
                      VT id = source[index];
                      if (id != source[index - 1]) { offsets[id] = index; }
                    });
-  ET zero = 0;
-  CUDA_TRY(cudaMemcpy(offsets, &zero, sizeof(ET), cudaMemcpyDefault));
+  thrust::device_ptr<VT> src = thrust::device_pointer_cast(source);
+  thrust::device_ptr<ET> off = thrust::device_pointer_cast(offsets);
+  off[src[0]]                = static_cast<ET>(0);
```

Comment:
Do you need a static_cast here?  Wouldn't `ET{0}` be sufficient?

Reply:
Sure. That can be fixed.